card-style.cls

```diff
@@ -36,6 +37,27 @@
 
 \newcounter{cards}
 
+\newcommand{\CardType}[1]{%
+  \def\cardcolor{}
+  \ifnum\pdfstrcmp{#1}{Item}=0      \def\cardcolor{blue} \fi
+  \ifnum\pdfstrcmp{#1}{Action}=0    \def\cardcolor{yellow} \fi
+  \ifnum\pdfstrcmp{#1}{Spell}=0     \def\cardcolor{green} \fi
+  \ifnum\pdfstrcmp{#1}{Cantrip}=0   \def\cardcolor{green} \fi
+  \ifnum\pdfstrcmp{#1}{Focus}=0     \def\cardcolor{green} \fi
```

Comment:
This looks like something `pgfkeys` can do for us, similar to `every ...` in tikz. Along these lines:

```latex
\tikzset{card background=line width=2mm}
\tikzset{Item card background=card background, color=blue}
...

\newcommand{\CardType}[1]{%
  \tikz[remember picture, overlay]{%
    \draw[#1 card background]
      ...
```

Reply:
Oh, yes that looks nice